src/main/java/com/shoppify/entity/Role.java

```diff
@@ -0,0 +1,33 @@
+package com.shoppify.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "ROLE")
+public class Role {
+   @Id
+   @GeneratedValue(strategy = GenerationType.IDENTITY)
+   @Column(name = "ID")
+   private long id;
+
+   @Column(name = "NAME")
+   private String name;
+
+   @Column(name = "DESCRIPTION")
+   private String description;
+
+   @ManyToOne
```

Comment:
remove @OnetoOne and change list<User> to Set<User> to ensure avoidance of duplication

Reply:
Một User sẽ có nhiều địa chỉ em